test/integration/authorization_test.go

```diff
@@ -289,10 +289,26 @@ func (test subjectAccessReviewTest) run(t *testing.T) {
 		}
 	}
 
-	if (actualResponse.Namespace != test.response.Namespace) ||
-		(actualResponse.Allowed != test.response.Allowed) ||
-		(!strings.HasPrefix(actualResponse.Reason, test.response.Reason)) {
-		t.Errorf("%s: from review\n\t%#v\nexpected\n\t%#v\ngot\n\t%#v", test.description, test.review, &test.response, actualResponse)
+	failMessage := ""
```

Comment:
> hmm, any way we can share or extend WaitForPolicyUpdate? would be nice if this code lived in one place

It can't reasonably be re-used.  `WaitForPolicyUpdate` is a clean way to wait until "Policy for me has changed to allow/disallow this", but it is not a clean way to test the SAR itself.  It would require making the API ugly for the majority of users, by forcing them to understand all the different options and permutations available for SARs and pass in a fully formed SAR request.